tensorflow/core/kernels/quantize_op_test.cc

```diff
@@ -14,37 +14,58 @@ limitations under the License.
 ==============================================================================*/
 
 #include <random>
+#include <stdexcept>
 
+#include "gtest/gtest-param-test.h"
```

Comment:
Rather than using gtest directly, we usually use `"tensorflow/core/platform/test.h"`, which _should_ wrap these.  You shouldn't need to include these headers.

Reply:
Removed. It was an artifact of clang-format on my end.